scss/util/_color.scss

```diff
@@ -23,7 +23,7 @@
     $rgb: nth($rgba, $i);
     $rgb: $rgb / 255;
 
-    $rgb: if($rgb < 0.03928, $rgb / 12.92, pow(($rgb + 0.055) / 1.055, 2.4));
+    $rgb: if($rgb < 0.03928, $rgb / 12.92, pow(($rgb + 0.055) / 1.055, 2.4, 18));
```

Comment:
I do not know how the precision parameter is used by pow and what would be the advantages and disadvantages of increasing it. All I can ask is "should we expect `row` to return an approximate value by default for this range of arguments). @andycochran ?